src/components/VImageDetails/VImageDetails.vue

```diff
@@ -26,9 +26,12 @@
       <div>
         <dt>{{ $t('image-details.information.source') }}</dt>
         <dd>
-          <VLink :href="image.foreign_landing_url" class="text-pink">{{
-            sourceName
-          }}</VLink>
+          <VLink
+            :href="image.foreign_landing_url"
+            @click="onSourceLinkClicked"
+            @keyup.enter="onSourceLinkClicked"
```

Comment:
Would this not send the event twice?

https://codesandbox.io/s/upbeat-wind-qllc8c?file=/src/App.vue

`@click` on a link should trigger both with <kbd>Enter</kbd> and a click.

Reply:
That seems to be the case! I had the same question because I remember seeing something about that once but I don't know exactly where and couldn't find it. Also, we are repeating these events in other places so I will leave only the clicks in those. Thank you!